src/collar/oc_particle.lsl

```diff
@@ -111,7 +114,7 @@ float g_fBurstRate = 0.0;
 list g_lCurrentChains = [];
 
 list g_lCollarPoints = [ // oc chain name, lockmeister name, lockguard name
-    "ooc"       , "collar"  , "collarfrontloop" , // Collar Front
+    "fcollar"       , "collar"  , "collarfrontloop" , // Collar Front
```

Comment:
Does this break compatibility with existing items?

Reply:
it did, but that should be fixed with line 166 -169. It renames the prim if needed.